base_layer/core/src/base_node/comms_interface/inbound_handlers.rs

```diff
@@ -343,7 +343,7 @@ where T: BlockchainBackend + 'static
                         target: LOG_TARGET,
                         "A peer has requested a block with hash {}", hash_hex,
                     );
-                    match self.blockchain_db.fetch_block_with_utxo(hash).await {
+                    match self.blockchain_db.fetch_block_with_transaction_output(hash).await {
```

Comment:
Is not reproducing the same results here, is this intentional? 
UTXO != TXO,  TXO = STXO || UTXO